Makefile

```diff
@@ -132,6 +132,7 @@ ifneq ($(wildcard .env.integration),)
 	$(HONCHO_COVERAGE_INTEGRATION) --pattern=integration_*.py
 else ifdef OPENSTACK_USER
 	echo -e "\nRunning integration tests with credentials from environment variables..."
+	echo -e "\n and by the way CONSUL_SERVERS is: --> $$CONSUL_SERVERS <--"
```

Comment:
@SSPJ This was for debugging, I don't think it's needed, could you remove it before merging?
If it's useful, maybe place it somewhere else or with another message.